src/test/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolverTest.scala

```diff
@@ -91,8 +91,8 @@ class InstanceUpdateOpResolverTest extends UnitTest with Inside {
         When("call taskTracker.task")
         verify(instanceTracker).instance(existingInstance.instanceId)
 
-        Then("result in an Expunge with the correct status")
-        stateChange shouldBe a[InstanceUpdateEffect.Expunge]
+        Then("result in an Update with the correct status")
+        stateChange shouldBe a[InstanceUpdateEffect.Update]
```

Comment:
Should we also assert the goal in the update?

Reply:
I don't think we change goal here...

Reply:
Why is there an update then?

Reply:
because we updated the task state, so we need to update the instance state. But we don't expunge, we keep it in state with `Goal = Running`